### DISCUSSION TITLE
CORS error

### DISCUSSION THREAD

**Discussion by scientist-v08** [vercel/community#4345] [Help] [answered]
### Summary

I'm trying to deploy my backend Node.JS on to Vercel but I'm getting a 404 error. I don't know the reason for the same. But it asked me to contact Vercel support if I am the site owner. 

### Example

https://www.manojvfullstackexample.in/api/details/read

### Steps to Reproduce

Just go to the above link and it will show a 404 error. 
https://github.com/manojvaadulasa/Backend. This is the git repo to my backend. I have no idea about what is wrong here.

**Comment by amyegan** [comment, marked as answer]
Hey @manojvaadulasa. I'm not sure the [404: NOT_FOUND](https://vercel.com/docs/errors/platform-error-codes#404:-not_found) error is related to CORS. These resources should help you get it working:

- [Using Express.js with Vercel](https://vercel.com/guides/using-express-with-vercel)
- [Cross-Origin Resource Sharing (CORS)](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS)
- [How can I enable CORS on Vercel?](https://vercel.com/guides/how-to-enable-cors)
- [Can I use Vercel as a reverse proxy?](https://vercel.com/guides/vercel-reverse-proxy-rewrites-external)
